components/runtime/vala/Makefile

```diff
@@ -45,6 +52,6 @@ install:	$(INSTALL_32_and_64)
 
 test:		$(TEST_32_and_64)
 
-BUILD_PKG_DEPENDENCIES =	$(BUILD_TOOLS)
-
-include ../../make-rules/depend.mk
+REQUIRED_PACKAGES += SUNWcs
```

Comment:
Are there build dependencies that are not auto-generated? (e.g. header-only packages)